recipe/meta.yaml

```diff
@@ -24,139 +20,159 @@ requirements:
+    - moto.ec2instanceconnect
     - moto.ecr
     - moto.ecs
+    - moto.efs
+    - moto.eks
+    - moto.elasticache
+    - moto.elasticbeanstalk
+    - moto.elastictranscoder
     - moto.elb
     - moto.elbv2
     - moto.emr
+    - moto.emrcontainers
+    - moto.es
     - moto.events
+    - moto.firehose
+    - moto.forecast
     - moto.glacier
     - moto.glue
+    - moto.guardduty
     - moto.iam
     - moto.instance_metadata
     - moto.iot
     - moto.iotdata
     - moto.kinesis
+    - moto.kinesisvideo
+    - moto.kinesisvideoarchivedmedia
     - moto.kms
     - moto.logs
+    - moto.managedblockchain
+    - moto.mediaconnect
+    - moto.medialive
+    - moto.mediapackage
+    - moto.mediastore
+    - moto.mediastoredata
     - moto.opsworks
     - moto.organizations
     - moto.packages
-    - moto.packages.httpretty
     - moto.polly
+    - moto.ram
     - moto.rds
-    - moto.rds2
     - moto.redshift
+    - moto.resourcegroups
     - moto.resourcegroupstaggingapi
     - moto.route53
+    - moto.route53resolver
     - moto.s3
     - moto.s3bucket_path
+    - moto.s3control
+    - moto.sagemaker
+    - moto.sdb
     - moto.secretsmanager
     - moto.ses
     - moto.sns
     - moto.sqs
     - moto.ssm
+    - moto.ssoadmin
+    - moto.stepfunctions
     - moto.sts
+    - moto.support
     - moto.swf
     - moto.swf.models
+    - moto.timestreamwrite
+    - moto.transcribe
+    - moto.utilities
+    - moto.wafv2
     - moto.xray
   requires:
-    - nose
-    - nose-exclude
-    - pytest
-    - pluggy <1.0.0
     - pip
-    - python <3.10
-    - freezegun
-    - parameterized
-    # needed for passing pip check
-    - rsa
-    - pyasn1
-  source_files:
-    - github/tests
   commands:
-    - pip check
+    # pip check fails on docker-py for windows
+    - pip check  # [not win]
     - moto_server --help
-    - pip install sure
-    - cd github
-#     - >
-#       pytest tests --verbose
-#         -k "not test_lambda"
-#         -k "not test_batch"
-#         -k "not test_lambda_cloudformation"
-#         -k "not test_cognitoidp.test_token_legitimacy"
-#         -k "not test_create_stack_lambda_and_dynamodb"
-#         -k "not tests.test_cloudformation.test_validate"
-#         -k "not test_cloudformation_stack_integration.test_lambda_function"
 
 about:
-  home: http://getmoto.org
-  license: Apache-2.0
+  home: https://docs.getmoto.org/en/latest/#
+  description : A library that allows you to easily mock out tests based on AWS infrastructure.
   summary: A library that allows your python tests to easily mock out the boto library.
+  license: Apache-2.0
   license_family: APACHE
   license_file: LICENSE
-  doc_url: http://docs.getmoto.org
+  license_url: https://github.com/spulec/moto/blob/master/LICENSE
+  doc_url: https://docs.getmoto.org
   dev_url: https://github.com/spulec/moto
 
 extra:
   recipe-maintainers:
+    - jan-janssen
     - johanneskoester
     - mariusvniekerk
     - hajapy
+
+  skip-lints:
+  - missing_pip_check
```

Comment:
Small nitpick:  `- missing_pip_check` could be indented. 

This doesn't need to block the merge.